scripts/build/p-klee-linux-ubuntu.inc

```diff
@@ -1,32 +1,35 @@
 install_build_dependencies_klee() {
-  apt update -y
-
-  # Add Kitware's certificate for CMake
-  dependencies=(
-    apt-transport-https
-    ca-certificates
-    gnupg
-    lsb-release
-    software-properties-common
-    wget
-  )
-
-  apt -y --no-install-recommends install "${dependencies[@]}"
-  wget -O - "https://apt.kitware.com/keys/kitware-archive-latest.asc" 2> /dev/null \
-      | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg > /dev/null
+  source "${DIR}/common-functions"
+
+  # Check if the Ubuntu version is an LTS version - otherwise no cmake version available
```

Comment:
Can you clarify that comment? Is this needed for 22.04 LTS?

Reply:
From https://github.com/klee/klee/pull/1493/commits/cba58ac3ee9753df60101669f306bed0bfa31f20:
>  Kitware's cmake is not available for non-LTS Ubuntu versions
>  Don't try to install it for non-LTS Ubuntu versions

Yes it's duplicated multiple times - but that the easiest way forward for now

Reply:
`p-cmake-linux-ubuntu.inc`?